week-3/Medha_Mathur/wallpaperChanger.py

```diff
@@ -0,0 +1,25 @@
+import struct
+import ctypes
+
+
+SPI_SETDESKWALLPAPER = 20
+WALLPAPER_PATH = 'C:\\Users\LENOVO\Desktop\weekly-challenges\week-3\Medha_Mathur\wallpaper.jpg'
```

Comment:
Don't hardcode absolute paths.
Use the `os` module. Example-
```suggestion
WALLPAPER_PATH = os.path.join(os.getcwd(), 'wallpaper.jpg')
```